src/ui/flutter_app/lib/models/preferences.dart

```diff
@@ -24,7 +24,15 @@ import 'package:sanmill/services/storage/adapters/algorithm_adapter.dart';
 
 part 'preferences.g.dart';
 
-enum Algorithms { AlphaBeta, PVS, MTD }
+@HiveType(typeId: 5)
+enum Algorithms {
+  @HiveField(0)
+  AlphaBeta,
+  @HiveField(1)
+  PVS,
+  @HiveField(2)
+  MTDF,
```

Comment:
MTDf

Reply:
enum types should actually be lowercase so it'll be `mtdf`. I just confused it with java where they should be uppercase :)